deploy/osd-delete-backplane-remediation-rbacs/20-delete-backplane-remediation-rbacs.CronJob.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: osd-delete-backplane-remediation-rbacs
+  namespace: openshift-backplane
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "*/10 * * * *"
```

Comment:
Is it a bit too frequent? If the expire time is 12h, I think we can run this clean up every several hours

Reply:
Hello, this is consistent with the period used between cleans for the cleanup of service accounts:
https://github.com/openshift/managed-cluster-config/blob/master/deploy/osd-delete-backplane-serviceaccounts/20-delete-backplane-serviceaccounts.CronJob.yaml#L10

That said, I agree, the job period could be 1 hour and still be acceptable